conan/internal/cache/cache.py

```diff
@@ -46,6 +46,12 @@ def temp_folder(self):
         # TODO: Improve the path definitions, this is very hardcoded
         return os.path.join(self._base_folder, "t")
 
+    @property
+    def filelock_folder(self):
+        """ Folder reserved for file locks on parallel operations
+        """
+        return os.path.join(self._base_folder, "l")
```

Comment:
Doens't need to be short name, as it will not contain large paths

Reply:
Done, commit d86b39cf0ad1bd1592082ecd05c6fbb25d14c694